platform/features/hyperlink/bundle.js

```diff
@@ -0,0 +1,133 @@
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+define([
+    './src/HyperlinkController',
+    'legacyRegistry',
+    'text!./res/templates/hyperlink.html'
+], function (
+    HyperlinkController,
+    legacyRegistry,
+    hyperlinkTemplate
+) {
+    legacyRegistry.register("platform/features/hyperlink", {
+        "name": "Hyperlink",
+        "description": "Insert a hyperlink to reference a link",
+        "extensions": {
+        "types": [
+            {
+                "key": "hyperlink",
+                "name": "Hyperlink",
+                "cssClass": "icon-page",
+                "description": "A hyperlink to redirect to a different link",
+                "features": ["creation"],
+                "properties": [
+                  {
+                      "key": "url",
+                      "name": "URL",
+                      "control": "textfield",
+                      "pattern": "^(ftp|https?)\\:\\/\\/",
+                      "required": true,
+                      "cssClass": "l-input-lg"
+                  },
+                  {
+                      "key": "displayText",
+                      "name": "Text to Display",
+                      "control": "textfield",
+                      "required": true,
+                      "cssClass": "l-input-lg"
+                  },
+                  {
+                      "key": "showTitle",
+                      "control": "checkbox",
+                      "value": false
+                  },{
+                      "key": "displayFormat",
+                      "name": "Display Format",
+                      "control": "composite",
+                      "items": [
+                          {
+                              "control": "select",
+                              "options": [
+                                  {
+                                      "name": "Link",
+                                      "value": "link"
+                                  },
+                                  {
+                                      "value": "button",
+                                      "name": "Button"
+                                  }
+                              ],
+                              "cssClass": "l-inline"
+                          }
+                      ]
+                  },
+                  {
+                      "key": "openNewTab",
+                      "name": "Tab to Open Hyperlink",
+                      "control": "composite",
```

Comment:
@sahajp23 Are you sure?  I checked out your code, make the change, and it works.  If it's not working for you then show me (screenshot / code) what you're seeing.